replica/compaction_group.hh

```diff
@@ -97,12 +97,6 @@ public:
     size_t memtable_count() const noexcept;
     // Returns minimum timestamp from memtable list
     api::timestamp_type min_memtable_timestamp() const;
-    // Add sstable to main set
-    // Exception safe
-    void add_sstable(sstables::shared_sstable sstable);
-    // Add sstable to maintenance set
-    // Exception safe
-    void add_maintenance_sstable(sstables::shared_sstable sst);
```

Comment:
This style of add-then-change-then-remove hinders review. The reviewer sees each change individually but it's hard to verify that "add" is equivalent to "remove". It results in smaller changes, but it's harder to see the overall picture.

Reply:
I can mention the eventual removal of those functions in the earlier patches to provide the context.
Otherwise, this can be done in the previous patch: `table: introduce table_sstables_adder`